setup_dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Following the structue:
+# -minecraft
+#    -- code
+#    -- data
+#    -- models
+
+mkdir -p ../data
+cd ../data
+
+wget https://craftassist.s3-us-west-2.amazonaws.com/pubr/dialogue_data.tar.gz
+wget https://craftassist.s3-us-west-2.amazonaws.com/pubr/instance_segmentation_data.tar.gz
+wget https://craftassist.s3-us-west-2.amazonaws.com/pubr/house_data.tar.gz
+
+tar -zxvf dialogue_data.tar.gz
+tar -zxvf instance_segmentation_data.tar.gz
+tar -zxvf house_data.tar.gz
+
+## For Ubuntu
+# Glog
+# https://codeyarns.com/2017/10/26/how-to-install-and-use-glog/
+sudo apt install libgoogle-glog-dev
```

Comment:
Is there a way to NOT require sudo permission in this script?

Reply:
I can remove sudo from `apt-get install`. Any suggestion otherwise?